code/modules/mob/living/carbon/shock.dm

```diff
@@ -104,23 +104,23 @@
 
 	sanity.onShock(shock_stage)
 
-	if(shock_stage == 10)
+	if(shock_stage == 30)
```

Comment:
You sure this is meant to be == and not >=
?

Reply:
That is exactly how Eris has it.
if(shock_stage == 30)